README.md

```diff
@@ -16,6 +16,85 @@ The MSAL library for Android gives your app the ability to use the [Microsoft Cl
 
 > Looking for developers interested in providing early feedback on a x-platform implementation of MSAL written in C++ and Java, callable from Java, Kotlin and C++.  If you're interested please please contact shoatman@microsoft.com.
 
+***2020-11-09*** New updates with [MSAL 2.0.3](https://github.com/AzureAD/microsoft-authentication-library-for-android/releases/tag/v2.0.3)
```

Comment:
I don't think all these updates are necessary.....just an overview of what is new with each major release should be sufficient.....this info is anyways available in the releases page as well as the changelog

Reply:
cut to 2.0.0